api/app/celery/research_mode_tasks.py

```diff
@@ -11,34 +11,26 @@
 from app.config import QueueNames
 from app.celery.process_ses_receipts_tasks import process_ses_results
 
-temp_fail = "7700900003"
-perm_fail = "7700900002"
-delivered = "7700900001"
+temp_fail = "409000003"
```

Comment:
Do you know what research mode is? I can see it's one of the modes a service can be placed under but there isn't documentation around it I could find.

Reply:
`research_mode` is a property on a service that can be enabled by platform admin.